provisioning/device/src/tpm_registration.ts

```diff
@@ -224,6 +227,15 @@ export class TpmRegistration extends EventEmitter implements RegistrationClient
 
   }
 
+  /**
+   * Sets the custom payload for registration.
```

Comment:
>for registration. [](start = 29, length = 17)

"that will be sent to the custom allocation policy implemented in an Azure Function (for more info see <link to docs>).